### PR TITLE
Feature/tg 347 add availability for accommodations

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs [hunk 1]
```diff
@@ -57,6 +57,19 @@ public async Task<IActionResult> ActivateAccommodations([FromBody] List<string>
...
     
+    /// <summary>
+    /// Mark all accommodations as IsAvailable
+    /// </summary>
+    /// <param name="cancellationToken">Cancellation token</param>
+    /// <returns></returns>
+    [HttpPost("activate/all")]
+    public async Task<IActionResult> ActivateAllAccommodations(CancellationToken cancellationToken)
+    {
+        await _accommodationService.SetAvailableAll(cancellationToken);
```
Comment (oleg-n-k) [inline HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs:68]: SetAllAvailable

File: HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs [hunk 2]
```diff
@@ -72,6 +85,19 @@ public async Task<IActionResult> DeactivateAccommodations([FromBody] List<string
...
+    
+    /// <summary>
+    /// Unmark all accommodations as IsAvailable
+    /// </summary>
+    /// <param name="cancellationToken">Cancellation token</param>
+    /// <returns></returns>
+    [HttpPost("deactivate")]
+    public async Task<IActionResult> DeactivateAllAccommodations(CancellationToken cancellationToken)
+    {
+        await _accommodationService.SetUnavailableAll(cancellationToken);
```
Comment (oleg-n-k) [inline HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs:98]: SetAllAccommodationsUnavailable

File: HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs]: Fixed

File: HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs (hunk 2 above)
Comment (DFSko) [reply HappyTravel.BaseConnector.Api/Controllers/AccommodationsController.cs]: Fixed

**Review by AlexeyShok — APPROVED** [review]

**Pull request merged**